dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogInterceptorTest.kt

```diff
@@ -191,7 +193,7 @@ internal class DatadogInterceptorTest : TracingInterceptorNotSendingSpanTest() {
             verify(rumMonitor.mockInstance).startResource(
                 requestId,
                 fakeMethod,
-                fakeUrl,
+                fakeUrl.lowercase(Locale.US),
```

Comment:
Because the regex for generating the host name in test changed to allow for upper case letters, and because of [RUMM-2900](https://datadoghq.atlassian.net/browse/RUMM-2900?atlOrigin=eyJpIjoiYTZhZjg0ZDMxODUwNDhlMmE0YzViN2RkYTExOWJjYTQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) we must currently lowercase the expected result.